packages/cubejs-docker/README.md

```diff
@@ -1,25 +1,105 @@
-- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) - Latest stable release on Alpine
-- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) - Latest development release from master branch.
+- [latest](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest.Dockerfile) -
+  Latest stable release, based on Debian (recommended)
+- [alpine](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/latest-alpine.Dockerfile) -
+  Latest stable release on Alpine
+- [dev](https://github.com/cube-js/cube.js/blob/master/packages/cubejs-docker/dev.Dockerfile) -
+  Latest development release from master branch.
+
+# Quick reference
+
+Where to get help? Check out our [Slack][link-slack], [Stack
+Overflow][link-stack-overflow] or [GitHub Discussions][link-github-discussions]
+pages.
+
+[link-slack]: https://slack.cube.dev/
+[link-stack-overflow]: https://stackoverflow.com/search?q=cube.js
+[link-github-discussions]: https://github.com/cube-js/cube.js/discussions
+
+Found an issue? File it on our GitHub Issues page.
+
+Supported architectures: `amd64`
+
+Need the documentation? Check out the [Cube.js documentation
+website][link-cubejs-docs].
+
+[link-cubejs-docs]: https://cube.dev/docs
 
 # What is Cube.js?
 
-Cube.js is an open-source analytical API platform. It is primarily used to build internal business intelligence tools or add customer-facing analytics to existing applications.
+Cube.js is an open-source analytical API platform. It is primarily used to build
+internal business intelligence tools or add customer-facing analytics to
+existing applications.
 
-Cube.js was designed to work with Serverless Query Engines like AWS Athena and Google BigQuery. Multi-stage querying approach makes it suitable for handling trillions of data points. Most modern RDBMS work with Cube.js as well and can be tuned for adequate performance.
+Cube.js was designed to work with Serverless Query Engines like AWS Athena and
+Google BigQuery. Multi-stage querying approach makes it suitable for handling
+trillions of data points. Most modern RDBMS work with Cube.js as well and can be
+tuned for adequate performance.
 
-Unlike others, it is not a monolith application, but a set of modules, which does one thing well. Cube.js provides modules to run transformations and modeling in data warehouse, querying and caching, managing API gateway and building UI on top of that.
+Unlike others, it is not a monolith application, but a set of modules, which
+does one thing well. Cube.js provides modules to run transformations and
+modeling in data warehouse, querying and caching, managing API gateway and
+building UI on top of that.
 
 # How to use this image
 
-```sh
+## Start a Cube.js instance
+
+```bash
 docker pull cubejs/cube:latest
+docker run -d -p 3000:3000 -p 4000:4000 \
+  -e CUBEJS_DB_HOST=postgres://username:password@hostname:port/dbname \
```

Comment:
It seems CUBEJS_DB_HOST contains not only the host in this example but a full DB URL string.